README.md

```diff
@@ -1,6 +1,11 @@
-# console
+[![crates.io](https://img.shields.io/crates/d/console.svg)](https://crates.io/crates/console)
+[![crates.io](https://img.shields.io/crates/v/console.svg)](https://crates.io/crates/console)
+[![Documentation](https://docs.rs/console/badge.svg)](https://docs.rs/console)
+[![Continuous Integration](https://github.com/mitsuhiko/console/workflows/Continuous%20Integration/badge.svg)](https://github.com/mitsuhiko/console/actions)
```

Comment:
I don't like to have these.

Reply:
All this, or just last?

Reply:
All this please

Reply:
Done.
But why? It is helpfull.

Reply:
I observed that it increases maintenance burden however infinitesimal it maybe.